dist/checkbox/ds4/checkbox.css

```diff
@@ -103,3 +121,27 @@
   fill: transparent;
   /* csslint ignore:end */
 }
+.checkbox__icon:not([hidden]) {
+  background: url('data:image/svg+xml,%3Csvg%20xmlns%3D%27http%3A//www.w3.org/2000/svg%27%20width%3D%2732%27%20height%3D%2732%27%3E%3Cpath%20%20fill%3D%22%23cccccc%22%20d%3D%22M25.107%2032.030h-18.214c-3.456%200-6.268-2.81-6.268-6.264v-19.529c0-3.456%202.812-6.268%206.268-6.268h18.214c3.456%200%206.268%202.812%206.268%206.268v19.529c0%203.452-2.812%206.264-6.268%206.264zM6.893%201.85c-2.419%200-4.386%201.967-4.386%204.386v19.529c0%202.417%201.967%204.382%204.386%204.382h18.214c2.419%200%204.386-1.965%204.386-4.382v-19.529c0-2.419-1.967-4.386-4.386-4.386h-18.214z%22%3E%3C/path%3E%3C/svg%3E');
```

Comment:
this can probably be externalized and inlined as a part of the build.

Reply:
@pranavjha This _is_ the build output. The LESS files hold the source of truth and this is the build output when used for static purposes (like the CDN).

Unless I'm missing what you're thinking of.

Reply:
There might be a way to shorten or simplify that encoded SVG, if that's what you mean. For example, I'm not sure if we need the full SVG xmlns definition URL in there. Anyhow, I will go ahead and merge and this is always something we can go ahead and improve later, or before release.

Reply:
@seangates, @ianmcburnie  Is there a less file that compiles to this css file?. Did not notice it in the PR. I am trying to understand the build setup.

Reply:
@seangates wrong placement (https://github.com/eBay/skin/pull/25/files#diff-b42df542824bb08ae25d4a684dbeaadcR95)

Reply:
Ah, that makes more sense, @pranavjha.